### PR TITLE
Configuration path improvements

### PR DESCRIPTION
**Pull request by mkurek**
- removed confusing field "path" from configuration module
- configuration class "**str**" is now only last module + class name
- renamed "name" field in ConfigurationClass to "class_name"
- added filtering by configuration module and all its descendants
- added filtering by configuration_path in API

### CODE REVIEW

File: src/ralph/admin/filters.py [hunk 1]
```diff
@@ -414,6 +414,54 @@ def queryset(self, request, queryset):
...
+class TreeRelatedAutocompleteFilterWithDescendants(
+    RelatedAutocompleteFieldListFilter
+):
+    """
+    Autocomplete filter for ForeignKeys to `mptt.models.MPTTModel` with
+    filtering by object and all its descendants.
+    """
+    def _get_descendants(self, request, root_id):
+        """
+        Return descendants of root object.
```
Comment (vi4m) [inline src/ralph/admin/filters.py]: Method descriptor already says that. Any more details? If not - remove.

File: src/ralph/admin/filters.py [hunk 2]
```diff
@@ -414,6 +414,54 @@ def queryset(self, request, queryset):
...
+    def queryset(self, request, queryset):
+        value = self.value()
+        if not value:
+            ids = []
+        else:
+            ids = value.split(',')
+        q_param = Q()
+        for id_ in ids:
+            if id_ == self.empty_value:
+                q_param |= Q(**{'{}__isnull'.format(self.field_path): True})
```
Comment (ar4s) [inline src/ralph/admin/filters.py:445]: Change this to `+` notation or change below to `.format`

Comment (mkurek) [inline src/ralph/admin/filters.py:445]: done

**Comment by ar4s** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**